### PR TITLE
Fixes #8960 - Handled ISE on bulk repos sync page

### PR DESCRIPTION
**Pull request by parthaa**
Syncing a repo with no feed url via the bulk sync page caused an ISE.
This has now been fixed.

### CODE REVIEW

**Comment by parthaa** [comment]
[test]

File: app/controllers/katello/api/v2/repositories_bulk_actions_controller.rb [hunk 1]
```diff
@@ -58,12 +58,17 @@ def destroy_repositories
     param :ids, Array, :desc => N_("List of repository ids"), :required => true
     def sync_repositories
       syncable_repositories = @repositories.syncable.has_url
+      if syncable_repositories.empty?
+        msg = _("Unable to synchronize any repository. You either do not have the permissions to"\
+                " synchronize or the selected repositories do not have a feed url.")
+        fail HttpErrors::NotFound, msg
```
Comment (ehelms) [inline app/controllers/katello/api/v2/repositories_bulk_actions_controller.rb]: I wonder if a 422 'Unprocessably Entity' might make more sense here.

File: app/controllers/katello/api/v2/repositories_bulk_actions_controller.rb [hunk 2]
```diff
@@ -58,12 +58,17 @@ def destroy_repositories
     param :ids, Array, :desc => N_("List of repository ids"), :required => true
     def sync_repositories
       syncable_repositories = @repositories.syncable.has_url
+      if syncable_repositories.empty?
+        msg = _("Unable to synchronize any repository. You either do not have the permissions to"\
```
Comment (ehelms) [inline app/controllers/katello/api/v2/repositories_bulk_actions_controller.rb]: Nit pick s/the permissions to/permission to

**Comment by parthaa** [comment]
[test]

**Comment by parthaa** [comment]
[test]

**Comment by ehelms** [comment]
ACK

**Pull request merged**